skaffold/src/main/resources/messages/SkaffoldBundle.properties

```diff
@@ -17,8 +17,8 @@
 skaffold.run.config.general.name=Skaffold
 skaffold.run.config.general.description=Runs Skaffold
 
-skaffold.run.config.single.run.name=Skaffold Single Run
-skaffold.run.config.dev.run.name=Skaffold Continuous
+skaffold.run.config.single.run.name=Skaffold Single Deployment
```

Comment:
How about just "Skaffold Deployment"

Reply:
or even more precisely: `Deploy to Kubernetes with Skaffold`? Seems like we have the real estate.

Reply:
Good idea, we probably should go straight to Kubernetes.
How about selection is "Kubernetes with Skaffold" -> opens up to two configurations "Deploy to Kubernetes with Skaffold" and "Develop on Kubernetes with Skaffold"?